core/logic/src/main/java/org/apache/syncope/core/logic/AbstractExecutableLogic.java

```diff
@@ -28,19 +26,22 @@
 import org.apache.syncope.common.rest.api.batch.BatchResponseItem;
 import org.apache.syncope.core.persistence.api.dao.search.OrderByClause;
 
+import java.util.Date;
```

Comment:
Please keep consistent import style (checkstyle definition was not aligned with `master`'s)